examples/features/sub-modules/wlp/readme.md

```diff
@@ -0,0 +1,11 @@
+# Apache ActiveMQ ResourceAdapter WLP / Openliberty Sample 
+
+This sample shows the usage of the Apache ActiveMQ Artemis ResourceAdapter with WLP / Openliberty.
+
+Usage:
+```bash
+mvn clean install
```

Comment:
Seems the example doesnt even suggest using the previously commented about script.